test.js

```diff
@@ -22,6 +22,11 @@ cleaner.clean(' foo\n', function (html) {
     assert.equal(html, 'foo');
 });
 
+// test that output is not trimmed in tags
+cleaner.clean('<p>I<strong> love</stong> Paris!</p>', function (html) {
```

Comment:
Just a note that there is a typo in line 26: `<strong> love</stong>` should be `</strong>`